packages/design-tokens/src/tokens/functional/components/river/base.json

```diff
@@ -1,32 +1,39 @@
 {
   "brand": {
     "River": {
-      "layout": {
+      "small": {
+        "padding": {
+          "value": "var(--base-size-28)"
+        },
+        "gap": {
+          "value": "var(--base-size-24)"
```

Comment:
Have you considered using our design tokens to manage responsiveness? We do that in [Grid](https://github.com/primer/brand/blob/main/packages/design-tokens/src/tokens/functional/components/grid/grid.json) for example.

Reply:
Good point, I just updated it to manage responsive values and looks so much clean and structured 👍